UANodeManager.cs

```diff
@@ -164,6 +170,63 @@ public override NodeId New(ISystemContext context, NodeState node)
             return new NodeId(Utils.IncrementIdentifier(ref _lastUsedId), (ushort)Server.NamespaceUris.GetIndex("http://opcfoundation.org/UA/EdgeTranslator/"));
         }
 
+        private void ImportFromBinary()
+        {
+            if (!_useWotNodesetBinary)
+            {
+                return;
+            }
+
+            NodeStateCollection predefinedNodes = new NodeStateCollection();
+            predefinedNodes.LoadFromBinaryResource(SystemContext, _wotNodesetBinary, this.GetType().GetTypeInfo().Assembly, true);
+
+            foreach (var node in predefinedNodes)
+            {
+                try
+                {
+                    AddPredefinedNode(SystemContext, node);
+                }
+                catch (Exception)
+                {
+                    // do nothing
+                }
+            }
+        }
+
+        public void ImportFromXml()
+        {
+            if (!_useWotNodeset)
+            {
+                return;

```

Comment:
Log an error here.

Reply:
This should never happen as File.Exists is already checked outside, I think it's better if I just remove it.